node/pkg/governor/manual_tokens.go

```diff
@@ -53,5 +53,13 @@ func manualTokenList() []tokenConfigEntry {
 		{chain: 37, addr: "00000000000000000000000074b7f16337b8972027f6196a17a631ac6de26d22", symbol: "USDC", coinGeckoId: "polygon-hermez-bridged-usdc-x-layer", decimals: 6, price: 0.9949},
 		{chain: 37, addr: "000000000000000000000000ea034fb02eb1808c2cc3adbc15f447b93cbe08e1", symbol: "WBTC", coinGeckoId: "polygon-hermez-bridged-wbtc-x-layer", decimals: 8, price: 57029},
 		{chain: 37, addr: "000000000000000000000000c5015b9d9161dca7e18e32f6f25c4ad850731fd4", symbol: "DAI", coinGeckoId: "polygon-hermez-bridged-dai-x-layer", decimals: 18, price: 1.0006},
+		// MANTLE (tokens over $50,000 24h volume)
+		{chain: 35, addr: "0000000000000000000000003c3a81e81dc49a522a592e7622a7e711c06bf354", symbol: "MNT", coinGeckoId: "mantle", decimals: 18, price: 1.01},
+		{chain: 35, addr: "00000000000000000000000078c1b0c915c4faa5fffa6cabf0219da63d7f4cb8", symbol: "WMNT", coinGeckoId: "wrapped-mantle", decimals: 18, price: 1.01},
+		{chain: 35, addr: "00000000000000000000000009bc4e0d864854c6afb6eb9a9cdf58ac190d0df9", symbol: "USDC", coinGeckoId: "mantle-bridged-usdc-mantle", decimals: 6, price: 1},
+		{chain: 35, addr: "000000000000000000000000201EBa5CC46D216Ce6DC03F6a759e8E766e956aE", symbol: "USDT", coinGeckoId: "mantle-bridged-usdt-mantle", decimals: 6, price: 0.9973},
+		{chain: 35, addr: "000000000000000000000000d5f7838f5c461feff7fe49ea5ebaf7728bb0adfa", symbol: "METH", coinGeckoId: "mantle-staked-ether", decimals: 18, price: 3934.06},
+		{chain: 35, addr: "000000000000000000000000deaddeaddeaddeaddeaddeaddeaddeaddead1111", symbol: "WETH", coinGeckoId: "wrapped-ether-mantle-bridge", decimals: 18, price: 3825.65},
+		{chain: 35, addr: "0000000000000000000000006e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd", symbol: "JOE", coinGeckoId: "joe", decimals: 18, price: 0.4911},
```

Comment:
Can you double check all these addresses please? Just checked JOE and it looks like you grabbed the wrong address from coingecko; you grabbed the Avalanche address rather than the Mantle address